cmd/rfdispatcher/rfserver.go

```diff
@@ -344,6 +344,7 @@ func main() {
 			for range ticker.C {
 				log.Trace("Running ticker")
 				server.rfd.RunPeriodic()
```

Comment:
REVIEWERS: added this to make it clear when we are in a ticker action and when we are finished with one.